dm/worker/task_checker.go

```diff
@@ -302,8 +300,10 @@ func (tsc *realTaskStatusChecker) getResumeStrategy(stStatus *pb.SubTaskStatus,
 	// TODO: use different strategies based on the error detail
 	for _, processErr := range stStatus.Result.Errors {
 		if !isResumableError(processErr) {
+			tsc.l.Info("error is not resumable", zap.Stringer("error", processErr))
```

Comment:
Maybe too many logs? but may not a problem.

Reply:
used to check task_checker does find an error and decide not to resume it. going to enable this only in failpoint.